lib/util/namedParameters.sh

```diff
@@ -100,23 +101,25 @@ Variable::TrapAssignNumberedParameter() {
       ;;
       'boolean')
         DEBUG Log passed "${!indirectAccess}", default "${__assign_varValue}"
+        local boolean_fingerprint="${__primitive_extension_fingerprint__boolean:+__primitive_extension_fingerprint__boolean:}"
```

Comment:
Shouldn't that last `:` be outside the variable definition `}`}? Unless I'm misunderstanding what happens here :)

Reply:
When using this file without the `util/type` module, the fingerprint is undefined. Thus, the "true" value is `:true` and the "false" value is `:false`. I thought it would be better to give to the user what they would expect, `true` and `false` instead of `:true`/`:false`. Do you see merit in leaving the colon?

Reply:
Ah, I get it now. Good thinking!